src/Javis.jl

```diff
@@ -941,8 +941,11 @@ function javis(
         ffmpeg_exe(`-loglevel panic -framerate $framerate -i $(tempdirectory)/%10d.png -i
                     "$(tempdirectory)/palette.bmp" -lavfi
                     "paletteuse=dither=sierra2_4a" -y $pathname`)
+    elseif ext == ".mp4"
+	ffmpeg_exe(`-loglevel panic -framerate $framerate -i $(tempdirectory)/%10d.png  -c:v libx264 -pix_fmt yuv420p $pathname`)
```

Comment:
this is still wrong indented on GitHub and in VSCode

Reply:
oh maybe you're aware of this :smile: . Thought you messaged me that it should be fine now.

Reply:
I think this is finally finally fixed! :joy: I checked it on Vim, VSCode, and plaintext! I firmly believe the indentation should be correct now!